userdocs/profiles.dev/src/components/HomepageFeatures.js

```diff
@@ -4,41 +4,38 @@ import styles from './HomepageFeatures.module.css';
 
 const FeatureList = [
   {
-    title: 'GitOps Native package manager',
-    Svg: require('../../static/img/undraw_docusaurus_mountain.svg').default,
+    title: 'What are Profiles?',
```

Comment:
The following line says "Profiles is ...", so this either needs to be:

> What is Profiles?

or

> What are 'Profiles'?

or neither

Reply:
fixed 👍🏻 , homepage is subject to change anyway